admin_guide/ipsec.adoc

```diff
@@ -53,21 +53,25 @@ required.
 After modifying the MTU in the {product-title} configuration, the SDN must be
 made aware of the change by removing the SDN interface and restarting the
 {product-title} node process.
-
-ifdef::openshift-enterprise[]
++
+.. Stop the node by removing the pod definition and rebooting the host:
++
 ----
-# systemctl stop atomic-openshift-node
-# ovs-vsctl del-br br0
-# systemctl start atomic-openshift-node
+# mkdir -p /etc/origin/node/pods-stopped
+# mv /etc/origin/node/pods/* /etc/origin/node/pods-stopped/
```

Comment:
- The static pod files are master services, not the node we want to reboot
- If we move files, we need to restore the files after that
- We don't need to reboot node services nor del-br br0 if we reboot the node at step 3

To fix, simply replace the steps with node reboot.